YOLayout/YOLayout_IMP.h

```diff
@@ -51,12 +51,8 @@ typedef enum {
   YOLayoutOptionsDefaultSize = YOLayoutOptionsDefaultHeight | YOLayoutOptionsDefaultWidth,
 
   // ALIGNMENT
-  //! After sizing, center vertically in the passed in rect
-  YOLayoutOptionsAlignCenterVertical = 1 << 10,
```

Comment:
I actually use both of these --sometimes I want to center horizontally but not vertically. Or sometimes I want to align right and center vertically. I feel pretty strongly about keeping separate vertical and horizontal centering options. Whats the reasoning for removing them?